src/NuGet.Indexing/Querys/QueryBoostingContext.cs

```diff
@@ -49,20 +49,15 @@ public static QueryBoostingContext Load(string fileName, ILoader loader, Framewo
 
                     var value = serializer.Deserialize<QueryBoostingContext>(reader);
```

Comment:
The search service will now fail without query boosting. Notice that for the unit tests I added this line:

```
 { "searchSettings.v1.json", JsonConvert.SerializeObject(QueryBoostingContext.Default) },
```

This adds the query boosting context to allow the unit tests to pass. Before, the tests passed even without the query boosting context.